modules/packages/CopyAggregation.chpl

```diff
@@ -368,8 +368,11 @@ module AggregationPrimitives {
   }
 
   proc getEnvInt(name: string, default: int): int {
-    extern proc getenv(name : c_string) : c_string;
-    var strval = getenv(name.localize().c_str()): string;
+    extern proc getenv(name : c_ptrConst(c_char)) : c_ptrConst(c_char);
+    const localName = name.localize();
+    const envValue = getenv(localName.c_str());
+    const len = __primitive("string_length_bytes", envValue);
```

Comment:
It seems unfortunate to use a primitive here (and in the other places). Could we add an unstable get-string-length to `CTypes` and use it here?

Reply:
added an unstable `strLen()` in `CTypes` to use here (and in other places).